src/auth/ui/vue/show.ts

```diff
@@ -236,7 +244,11 @@ export class AuthWebview extends VueWebview {
                 return { id: 'badStartUrl', text: `Connection failed. Please verify your start URL.` }
             }
 
-            getLogger().error('AuthWebview: Failed to setup: %s', (e as Error).message)
+            // If SSO setup fails we want to be able to show the user an error in the UI, due to this we cannot
```

Comment:
`handleError` logs the error, right? so that's why we can remove the getLogger call here

Reply:
Yeah, handleError logs the error + shows a message to the user. I've renamed the method to make it more clear.